### DISCUSSION TITLE
GitHub CoPilot has not been started

### DISCUSSION THREAD

**Discussion by oliverschloebe** [community/community#65880] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

The GitHub Copilot extension for VS Code isn't even starting. When viewing the extension's runtime status it's saying "Has not been started". I'm not getting any output in the Output panel for "Copilot" either so I can't provide any error logs.

The Copilot icon is visible in the VS Code status bar but it's not doing anything either.

I'm running VS Code 1.81.1 and Copilot extension v1.108.382

**Comment by arafajs** [comment]
- Make sure you have signed in to the extension with your GitHub account. The extension needs to authenticate with GitHub to function.
- Try restarting VS Code. Sometimes extensions can get in a funky state and a restart clears things up.
- Try disabling other extensions to see if there are conflicts. Especially look out for other AI assistant extensions.
- Remove the Copilot extension and reinstall a fresh copy from the Marketplace.
- Open the Copilot output channel (View > Output > Copilot) and see if any errors are being logged there.
- Update your graphics drivers if you're on Windows. Outdated drivers can cause issues.
- Make sure your internet connection is stable. The extension needs an internet connection to call the Copilot cloud APIs.
- Try installing the Insiders version of the Copilot extension to see if a newer build fixes it.

**Comment by oliverschloebe** [comment]
Hey @arafajs, thanks for your quick reply!

I did all of your mentioned suggestions and it's still not working at all. I'm even still not getting output in the **Copilot** output channel, only in the **GitHub Authentication** output channel which reads

> 2023-09-04 12:42:28.580 [info] Reading sessions from keychain...
> 2023-09-04 12:42:28.580 [info] Getting sessions for all scopes...
> 2023-09-04 12:42:28.580 [info] Got stored sessions!
> 2023-09-04 12:42:28.580 [info] Got 1 verified sessions.
> 2023-09-04 12:42:28.581 [info] Got 1 sessions for ...
> 2023-09-04 12:42:28.581 [info] Getting sessions for all scopes...
> 2023-09-04 12:42:28.581 [info] Got 1 sessions for ...
> 2023-09-04 12:42:28.590 [info] Getting sessions for user:email...
> 2023-09-04 12:42:28.590 [info] Got 1 sessions for user:email...
> 2023-09-04 12:42:28.959 [info] Getting sessions for all scopes...
> 2023-09-04 12:42:28.960 [info] Got 1 sessions for ...

What else can I do?

**Reply by arafajs** [reply]
- Make sure you don't have any proxy configurations or VPNs active that could be interfering with the extension calling out to GitHub's APIs. Try disabling any proxies temporarily to see if that helps.
- Check if you have any firewall or antivirus software that could be blocking the extension. Try adding exceptions for VS Code and the Copilot extension.
- Try using GitHub Copilot in another editor that supports it like Neovim or JetBrains IDEs. See if the issue is isolated to VS Code.
- As a last resort, you can try uninstalling and reinstalling VS Code completely to rule out any corrupted files.
- Also remove any existing VS Code extensions, settings, and cache files before reinstalling.

If it's still not working after trying everything, I would recommend contacting GitHub Support directly to troubleshoot further since it seems to be something environmental or a bug. Make sure to include all the debugging details and logs you have gathered.

**Comment by oliverschloebe** [comment, marked as answer]
Thank you, tested in PHPStorm and it worked just fine.

Then made a fresh VS Code install and it turned out that the `Remote FS` extension is causing the issue. I just created a ticket here: https://github.com/liximomo/vscode-remote-fs/issues/113

Thanks for your feedback!
